api/src/main/java/com/datatorrent/api/DefaultInputPort.java

```diff
@@ -55,6 +55,15 @@ public void setConnected(boolean connected)
     this.connected = connected;
   }
 
+  /**
+   * Returns True if this port is connected. False otherwise.
```

Comment:
@ilooner sorry about nitpicking. The return type is a primitive boolean so can you please change `True` to `true` and `False` to `false`.